src/wireviz/wv_colors.py

```diff
@@ -19,10 +43,17 @@
     'TQ': '#00ffff',
     'BU': '#0066ff',
     'VT': '#8000ff',
-    'BN': '#666600',
+    'BN': '#a52a2a',
```

Comment:
Not a huge fan of the new brown color, it looks too similar to orange, see example 01

<img width="400" alt="Screen Shot 2020-07-01 at 21 03 17" src="https://user-images.githubusercontent.com/2447251/86281779-4c921f80-bbde-11ea-9660-5e43ba23bba7.png">